string csv_path
---
float32[] tracking_sequence
---
float32[] time_sequence